docs/source/conf.py

```diff
@@ -331,6 +331,7 @@
     ("py:class", "matplotlib.figure.Figure"),
     ("py:class", "plotly.graph_objects.Figure"),
     ("py:class", "PIL.Image.Image"),
+    ("py:class", "mlflow.deployments.base.BaseDeploymentClient"),
```

Comment:
This is to avoid this error when building the documentation:
```
/home/teppi/mlflow/mlflow/sagemaker/__init__.py:docstring of mlflow.sagemaker.SageMakerPlugin:1:py:class reference target not found: mlflow.deployments.base.BaseDeploymentClient
```

Reply:
I believe it comes from this line: https://github.com/mlflow/mlflow/blob/bacc5d5aa903a9c9484ca75c218818159f8e641c/mlflow/sagemaker/__init__.py#L1748

The error goes away if BaseDeploymentClient is not declared as the parent class.

Reply:
Got it! Thanks for clarifying :)